src/frontend/qt_sdl/PlatformConfig.cpp

```diff
@@ -119,6 +120,7 @@ ConfigEntry PlatformConfigFile[] =
     {"HKJoy_Reset",               0, &HKJoyMapping[HK_Reset],               -1, NULL, 0},
     {"HKJoy_FastForward",         0, &HKJoyMapping[HK_FastForward],         -1, NULL, 0},
     {"HKJoy_FastForwardToggle",   0, &HKJoyMapping[HK_FastForwardToggle],   -1, NULL, 0},
+    {"HKJoy_FastForwardToggle",   0, &HKJoyMapping[HK_FullscreenToggle],   -1, NULL, 0},
```

Comment:
This is misaligned because of a missing space.

Reply:
done